proto/skaffold.proto

```diff
@@ -231,4 +231,25 @@ enum ErrorCode {
 //    COULD_NOT_GET_DOCKER_CLIENT = 7;
 //    COULD_NOT_GET_LOCAL_CLUSTER = 8;
 //    BUILDER_CLEANUP = 9;
+
+    STATUS_CHECK_NO_ERROR = 300;
+    STATUS_CHECK_UNKNOWN = 301;
+    STATUS_CHECK_UNKNOWN_UNSCHEDULABLE = 302;
+
```

Comment:
let's add docs for each of these, so they show up nicely in the generated docs http://34.94.185.45:1313/docs/references/api/grpc/#errorcode